src/main/java/co/org/osso/api/BookController.java

```diff
@@ -0,0 +1,38 @@
+package co.org.osso.api;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api")
+public class BookController {
+
+    private final CustomerService customerService;
+    private final BookService bookService;
+
+    BookController(CustomerService customerService, BookService bookService){
+        this.customerService = customerService;
+        this.bookService = bookService;
+    }
+
+    @GetMapping("/customer")
+    public List getCustomers() {
+        return customerService.getCustomers();
+    }
+
+    @PostMapping("/book")
+    @ResponseStatus(HttpStatus.CREATED)
+    public Book postData(
```

Comment:
The parameter list can be on one line here. Also, can we rename this method to something more descriptive.

```suggestion
    public Book postBook(@RequestBody final Book book) {
```

BTW: ^This^ is a new feature in GitHub I'm trying out. I don't think you should commit this, because it only applies to one line, and will probably not compile if you apply it. But hopefully it shows you what I mean.

Reply:
Done